protocol/testutil/keeper/prices.go

```diff
@@ -258,3 +266,9 @@ func CreateTestPriceMarkets(
 		marketId++
 	}
 }
+
+func GetNumMarkets(t *testing.T, ctx sdk.Context, keeper *keeper.Keeper) uint32 {
```

Comment:
Moved this method from the keeper struct since protocol logic *should not* rely on number of markets of anything, now that we've removed consecutive ID requirement